### PR TITLE
Docs: migrating the usage instructions to appland.com/docs

### PR DESCRIPTION
**Pull request by ptrdvrk**
Removing the usage instructions and adding a link to the doc site.

### CODE REVIEW

**Review by apotterri — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -378,7 +58,8 @@ MANIFEST.in	README.rst	blog		setup.py	tddium.yml	tox.ini
 ## Python version support
 As a package intended to be installed in as many environments as possible, `appmap-python`
 needs to avoid using features of Python or the standard library that were added after the
-oldest version currently supported (see [above](#supported-versions)).
+oldest version currently supported (see the
+[supported versions](https://appland.com/docs/reference/appmap-python.html#supported-versions)).
```
[the remark this answers is not in the document]
Comment (ptrdvrk) [reply README.md:62]: My expectation is that a vast majority of the python users will read the documentation on the doc site and only those who want to see the sources or contribute will access this README and other resources, so I would prefer to keep the supported versions in the documentation site.

[the remark this answers is not in the document]
Comment (apotterri) [reply README.md:62]: Ok. Then can you add a comment to https://github.com/applandinc/appmap-python/blob/master/pyproject.toml#L34 , please? It should have a link the source for the doc site where the versions are specified, with a reminder that changing the supported versions will also require a doc update.

[the remark this answers is not in the document]
Comment (ptrdvrk) [reply README.md:62]: Done, thank you for the review Alan!

**Comment by apotterri** [comment]
Looks good, thanks.

**Review by apotterri — APPROVED** [review]

**Pull request merged**